modules/optimizing-storage-azure.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: REFERENCE
```

Comment:
Please add the rest of the [module metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata).

Reply:
Resolved, thanks